server/odc-service/src/main/java/com/oceanbase/odc/service/onlineschemachange/DefaultOnlineSchemaChangeTaskHandler.java

```diff
@@ -468,4 +469,29 @@ private void dropNewTableIfExits(OnlineSchemaChangeScheduleTaskParameters taskPa
         }
     }
 
+    /**
+     * parse and set compatible fields
+     * 
+     * @param jsonStr
+     * @return
+     */
+    private OnlineSchemaChangeParameters parseOnlineSchemaChangeParameters(String jsonStr) {
+        OnlineSchemaChangeParameters onlineSchemaChangeParameters = JsonUtils.fromJson(
+                jsonStr, OnlineSchemaChangeParameters.class);
+        // correct null value to default RateLimiterConfig object
+        if (null == onlineSchemaChangeParameters.getRateLimitConfig()) {
+            onlineSchemaChangeParameters.setRateLimitConfig(new RateLimiterConfig());
+        }
+        return onlineSchemaChangeParameters;
+    }
+
+    private OnlineSchemaChangeScheduleTaskParameters parseOnlineSchemaChangeScheduleTaskParameters(String jsonStr) {
+        OnlineSchemaChangeScheduleTaskParameters OnlineSchemaChangeScheduleTaskParameters = JsonUtils.fromJson(
```

Comment:
The naming of variables should use lower camera case

Reply:
fixed